Interfaces/GCHP/gchp_chunk_mod.F90

```diff
@@ -1134,4 +1571,97 @@ SUBROUTINE GCHP_Chunk_Run( GC,                                             &
 
   END SUBROUTINE GCHP_Chunk_Run
 !EOC
+
+!BOP
+  SUBROUTINE GCHP_PRINT_MET(I, J, L,         &
```

Comment:
This could be reused for GC-Classic and stored in a more generic module. Can keep for now and I'll move it later.